src/main/resources/static/templates/CSRFVulnerability/LEVEL_4/CSRFVulnerability.js

```
@@ -0,0 +1,31 @@
+    dataPost.append("newpass", "");
+    dataPost.append("rtkparam", "");
+    doPostAjaxCall(CallbackResponse, url, true, dataPost);
+  });
+}
+addingEventListenerToSendButton();
+
+function addingEventListenerToResetPassButton() {
+  document
+    .getElementById("resetPassBtn")
+    .addEventListener("click", function () {
+      let url = getUrlForVulnerabilityLevel();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'url' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)